tests/test_invenio_stats.py

```diff
@@ -100,5 +103,43 @@ def test_batch_events(app, event_entrypoints, objects,
 
 
 def test_wrong_intervals(app):
+    """Test wrong interval error."""
     with pytest.raises(ValueError):
         StatAggregator(current_search_client, 'test', 'test', 'month', 'day')
+
+
+def test_overwriting_aggregations(app, mock_user_ctx, sequential_ids):
+    """Test that new aggregations in the same intervals overwrite old."""
```

Comment:
As this test is quite complex we could add some comments explaining how it works.